src/components/About/About.module.css

```diff
@@ -1,3 +1,176 @@
 .title {
   text-align: center;
 }
+
+@keyframes ldio {
+
+  0% {
+    opacity: 1;
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+.ldio div {
+  left: 94px;
+  top: 48px;
+  position: absolute;
+  animation: ldio linear 1s infinite;
+  background: #fe718d;
+  width: 12px;
+  height: 24px;
+  border-radius: 6px / 12px;
+  transform-origin: 6px 52px;
+}
+
+.ldio div:nth-child(1) {
+  transform: rotate(0deg);
+  animation-delay: -.9166666666666666s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(2) {
+  transform: rotate(30deg);
+  animation-delay: -.8333333333333334s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(3) {
+  transform: rotate(60deg);
+  animation-delay: -.75s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(4) {
+  transform: rotate(90deg);
+  animation-delay: -.6666666666666666s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(5) {
+  transform: rotate(120deg);
+  animation-delay: -.5833333333333334s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(6) {
+  transform: rotate(150deg);
+  animation-delay: -0.5s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(7) {
+  transform: rotate(180deg);
+  animation-delay: -.4166666666666667s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(8) {
+  transform: rotate(210deg);
+  animation-delay: -.3333333333333333s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(9) {
+  transform: rotate(240deg);
+  animation-delay: -.25s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(10) {
+  transform: rotate(270deg);
+  animation-delay: -.16666666666666666s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(11) {
+  transform: rotate(300deg);
+  animation-delay: -.08333333333333333s;
+  background: #fe718d;
+}
+  
+.ldio div:nth-child(12) {
+  transform: rotate(330deg);
+  animation-delay: 0s;
+  background: #fe718d;
+}
+
+.loadingio {
+  width: 200px;
+  height: 200px;
+  display: inline-block;
+  overflow: hidden;
+  background: #fff;
+}
+
+.ldio {
+  width: 100%;
+  height: 100%;
+  position: relative;
+  transform: translateZ(0) scale(1);
+  backface-visibility: hidden;
+  transform-origin: 0 0;
+}
+
+.ldio div {
```

Comment:
Codacy found an issue: [Expected selector ".ldio div" to come before selector ".ldio div:nth-child(1)" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5763907)